src/recon.rs

```diff
@@ -1945,7 +1945,7 @@ pub(crate) unsafe fn rav1d_read_coef_blocks<BD: BitDepth>(
                                 (*ts).msac.rng,
                             );
                         }
-                        cbi[t.bx as usize].txtp[0] = txtp as u8;
+                        f.frame_thread.cbi[cbi_idx..][t.bx as usize].txtp[0] = txtp as u8;
```

Comment:
Can this and the above index be combined into one lookup?  With `debug_block_info!` as a macro now, I don't think it'll cause any issues.

Same for the other pair below.